### PR TITLE
Added use-interval package, replaced useEffect in index.js with custom useInterval hook

### PR DESCRIPTION
**Pull request by corquaid**
@tammaroivan 

Hi Ivan, 

I used your night-region package in one of my projects and modified it to live-update using a custom hook that incorporates setInterval, which is tricky to use with the stock useEffect hook.

I noticed that a live update was on your TODOs for the package so this pull request updates your code replaces the index.js useEffect hook with the custom useInterval hook. Just to show my appreciation for your super-useful code!

Here's a quick screen capture of the live-updating night-region running on my app.

https://user-images.githubusercontent.com/69508715/131838429-0bbe5aa6-54ff-4e4a-b640-211311529016.mp4

Please feel free to review and merge if you're happy to do so.

Thanks!
Cormac

### CODE REVIEW

**Comment by tammaroivan** [comment]
Hey @corquaid, thank you for this! I really appreciate it. 

What do you think if we make the 3000ms in the useInterval a prop with that value as default so anyone can customize after how many ms to refresh it? I think that can be useful. 

If you have some time, if not I can merge like this and modify it later.

Thanks!

**Comment by corquaid** [comment]
Leave it with me!

File: src/index.js [hunk 1]
```diff
@@ -34,15 +37,10 @@ export const NightRegion = (props) => {
     return latLng
   }
 
-  useEffect(() => {
+  useInterval(async () => {
```
Comment (tammaroivan-kantox) [inline src/index.js:40]: Does the arrow function need to be async?

Comment (tammaroivan) [inline src/index.js]: Does the arrow function need to be async?

Comment (corquaid) [reply src/index.js]: No doesn't look like it. Not sure how that got in but fixing now.

**Review by tammaroivan-kantox — APPROVED** [review]

**Review by tammaroivan — APPROVED** [review]

**Comment by tammaroivan** [comment]
Thanks again @corquaid!

**Comment by corquaid** [comment]
> Thanks again @corquaid!

No problem, glad to contribute.

**Pull request merged**